
Loading components for TIF035

Traceback (most recent call last):
  File "./bulkAl.py", line 31, in <module>
Traceback (most recent call last):
  File "./bulkAl.py", line 31, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
  File "./bulkAl.py", line 31, in <module>
Traceback (most recent call last):
  File "./bulkAl.py", line 31, in <module>
Traceback (most recent call last):
  File "./bulkAl.py", line 31, in <module>
Traceback (most recent call last):
  File "./bulkAl.py", line 31, in <module>
Traceback (most recent call last):
  File "./bulkAl.py", line 31, in <module>
  File "./bulkAl.py", line 31, in <module>
    energy = bulk.get_potential_energy()
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
    energy = bulk.get_potential_energy()
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
    energy = bulk.get_potential_energy()
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
    energy = bulk.get_potential_energy()
    return self._calc.get_potential_energy(self)
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
    energy = bulk.get_potential_energy()
    return self._calc.get_potential_energy(self)
    energy = bulk.get_potential_energy()
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
    energy = bulk.get_potential_energy()
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
    self.calculate(atoms, converge=True)
    energy = bulk.get_potential_energy()
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
  File "/beda/sw/apps/ase/r2828/lib/python2.6/site-packages/ase/atoms.py", line 627, in get_potential_energy
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 229, in calculate
    return self._calc.get_potential_energy(self)
    self.calculate(atoms, converge=True)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 229, in calculate
    return self._calc.get_potential_energy(self)
    self.calculate(atoms, converge=True)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 229, in calculate
    return self._calc.get_potential_energy(self)
    self.calculate(atoms, converge=True)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
    return self._calc.get_potential_energy(self)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 229, in calculate
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/aseinterface.py", line 38, in get_potential_energy
    self.initialize(atoms)
    self.calculate(atoms, converge=True)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 505, in initialize
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 229, in calculate
    self.initialize(atoms)
    self.calculate(atoms, converge=True)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 505, in initialize
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 229, in calculate
    self.initialize(atoms)
    self.calculate(atoms, converge=True)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 505, in initialize
    self.initialize(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 229, in calculate
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 505, in initialize
    self.calculate(atoms, converge=True)
    bd = BandDescriptor(nbands, band_comm, parstride_bands)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 229, in calculate
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/band_descriptor.py", line 90, in __init__
    self.initialize(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 505, in initialize
    bd = BandDescriptor(nbands, band_comm, parstride_bands)
    self.initialize(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/band_descriptor.py", line 90, in __init__
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 505, in initialize
    bd = BandDescriptor(nbands, band_comm, parstride_bands)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/band_descriptor.py", line 90, in __init__
    self.initialize(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 505, in initialize
    bd = BandDescriptor(nbands, band_comm, parstride_bands)
    self.initialize(atoms)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/band_descriptor.py", line 90, in __init__
    (self.nbands, self.comm.size))
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/paw.py", line 505, in initialize
RuntimeError: Cannot distribute 108 bands to 8 processors
GPAW CLEANUP (node 0): <type 'exceptions.RuntimeError'> occurred.  Calling MPI_Abort!
    bd = BandDescriptor(nbands, band_comm, parstride_bands)
    (self.nbands, self.comm.size))
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/band_descriptor.py", line 90, in __init__
RuntimeError: Cannot distribute 108 bands to 8 processors
GPAW CLEANUP (node 7): <type 'exceptions.RuntimeError'> occurred.  Calling MPI_Abort!
    bd = BandDescriptor(nbands, band_comm, parstride_bands)
    (self.nbands, self.comm.size))
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/band_descriptor.py", line 90, in __init__
RuntimeError: Cannot distribute 108 bands to 8 processors
GPAW CLEANUP (node 5): <type 'exceptions.RuntimeError'> occurred.  Calling MPI_Abort!
    bd = BandDescriptor(nbands, band_comm, parstride_bands)
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/band_descriptor.py", line 90, in __init__
    (self.nbands, self.comm.size))
RuntimeError: Cannot distribute 108 bands to 8 processors
GPAW CLEANUP (node 6): <type 'exceptions.RuntimeError'> occurred.  Calling MPI_Abort!
        (self.nbands, self.comm.size))
bd = BandDescriptor(nbands, band_comm, parstride_bands)
    (self.nbands, self.comm.size))
RuntimeError: Cannot distribute 108 bands to 8 processors
GPAW CLEANUP (node 3): <type 'exceptions.RuntimeError'> occurred.  Calling MPI_Abort!
    (self.nbands, self.comm.size))
  File "/beda/sw/apps/GPAW/r9737-py26-i111-mkl102/lib/python2.6/site-packages/gpaw/band_descriptor.py", line 90, in __init__
RuntimeError: Cannot distribute 108 bands to 8 processors
GPAW CLEANUP (node 2): <type 'exceptions.RuntimeError'> occurred.  Calling MPI_Abort!
RuntimeError: Cannot distribute 108 bands to 8 processors
    (self.nbands, self.comm.size))
GPAW CLEANUP (node 1): <type 'exceptions.RuntimeError'> occurred.  Calling MPI_Abort!
RuntimeError: Cannot distribute 108 bands to 8 processors
GPAW CLEANUP (node 4): <type 'exceptions.RuntimeError'> occurred.  Calling MPI_Abort!
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD 
with errorcode 42.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun has exited due to process rank 0 with PID 30493 on
node beda63-1.nfs.private exiting without calling "finalize". This may
have caused other processes in the application to be
terminated by signals sent by mpirun (as reported here).
--------------------------------------------------------------------------
[beda63-1.nfs.private:30492] 7 more processes have sent help message help-mpi-api.txt / mpi-abort
[beda63-1.nfs.private:30492] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
